code/modules/hydroponics/grown/nymph.dm

```diff
@@ -12,6 +12,37 @@
 	yield = 1
 	reagents_add = list("plantmatter" = 0.1)
 
+// If the nymph pods are fully grown, allow ghosts that click on them to spawn as a nymph (decreases yield by 1 and kills the plant when yield reaches 0)
+/obj/item/seeds/nymph/GhostAttackWhenPlanted(mob/dead/observer/O, obj/machinery/hydroponics/T)
+	if(!T.harvest) 
+		to_chat(O, "[src] is not yet ready.")
+		return
+	if(T.dead)
+		to_chat(O, "[src] is dead!")
+		return
+	if(!(O in GLOB.respawnable_list))
+		to_chat(O, "You are not permitted to rejoin the round.")
+		return
+	else if(cannotPossess(O))
+		to_chat(O, "You have enabled antag HUD and are unable to re-enter the round.")
+		return
+	var/nymph_ask = alert("Become a Diona Nymph? You will not be able to be cloned!", "Diona Nymph Pod", "Yes", "No")
+	if(nymph_ask == "No" || !src || QDELETED(src))
+		return
+	if(T.myseed && yield > 0 && !T.dead)
+		yield -= 1
+		var/mob/living/simple_animal/diona/D = new /mob/living/simple_animal/diona(get_turf(T))
+		D.key = O.key
+		GLOB.respawnable_list -= O
+		visible_message(D, "A new diona nymph emerges from the pod, its antennae waving excitedly.")
+		if(yield <= 0)
+			visible_message("The seed pod withers away, now merely an empty husk.")
+			T.plantdies()
+		spawn(5)
```

Comment:
...why?

Reply:
I'm not following what you're referring to, sorry.

Reply:
Why does `GLOB.respawnable_list += usr` need to be in a spawn with a 5 second delay?

Reply:
Because that's the way Respawn As Mob did it.

Reply:
Because that's the way Respawn As Mob did it.